roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -149,12 +151,31 @@
   failed_when: >
     reposub.rc != 0 or 'error: ' in reposub.stdout.lower()
   changed_when: reposub.rc == '0'
-  when: >
-    tpa_subscription_token|length > 0
-    and install_bdr_packages|bool
-    and pg_version|int >= 14
+  when:
+    - enable_edb_repo|bool
+    - tpa_subscription_token|length > 0
+    - install_bdr_packages|bool
+    - pg_version|int == 14
+    - bdr_version|int == 4
   loop: "{{ edb_2q_repositories }}"
 
+- name: Install PGD RPM packages repo
+  ansible.builtin.yum_repository:
+    name: "{{ item.name }}"
+    description: "{{ item.description }}"
+    file: "enterprisedb-postgres_distributed"
```

Comment:
Can both repositories have the same name? Wouldn't this create two files with `enterprisedb-postgres_distributed.repo`? Maybe use the `{{ item.name }}` value?

Reply:
If the `file` parameter is not included, the module defaults to `name`, so removal of this parameter would be sufficient.